src/components/ebay-fake-tabs/examples/02-fake-with-starting-index/template.marko

```diff
@@ -0,0 +1,8 @@
+<ebay-fake-tabs selected-index==1>
```

Comment:
Typo I think (double equals). Also make sure to update the ebay-tabs with selected-index too.